src/transformers/onnx/features.py

```diff
@@ -78,6 +79,7 @@ class FeaturesManager:
             "sequence-classification-with-past",
             onnx_config_cls=GPTNeoOnnxConfig,
         ),
+        "layoutlm": supported_features_mapping("default", onnx_config_cls=LayoutLMOnnxConfig),
```

Comment:
Is there any limitation that makes support for masked language model, sequence classification and token classification tasks complicated?

If that's the case, then we can go forward and add support for these tasks later, but if not you can add those now:

```suggestion
        "layoutlm": supported_features_mapping(
            "default", 
            "masked-lm",
            "sequence-classification",          
            "token-classification",                               
            onnx_config_cls=LayoutLMOnnxConfig,                  
        )   
```
This also means adding a key "masked-lm" mapping to the AutoModelForMaskedLM class to the _TASK_TO_AUTOMODELS dict.

Reply:
I don't see any limitation that would complicate those tasks. I think we can add them now. @NielsRogge what do you think?

Reply:
Yes you can add them.